main.tf

```diff
@@ -129,6 +130,12 @@ resource "aws_codepipeline" "pipeline" {
       stage[0].action[0].configuration // ignore GitHub's OAuthToken
     ]
   }
+
+  tags = {
+    env = var.env_tag
+    data-sensitivity = var.data_sensitivity_tag
+    repo = "https://github.com/byu-oit/${var.repo_name}"
+  }
```

Comment:
You may want to consider using a local for tags, so that you don't have to repeat this.
At the root:
```
locals {
  tags = {
    env = var.env_tag
    data-sensitivity = var.data_sensitivity_tag
    repo = "https://github.com/byu-oit/${var.repo_name}"
  }
```
In your resources:
```
  tags = local.tags
```